frontend/src/components/Listing/listing.jsx

```diff
@@ -2,62 +2,273 @@ import React, { Component } from "react";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
 import {
-  setPrice,
-  setBedRoom,
-  setWashRoom,
-  setLaundryRoom,
-  setDescription,
-  setImage
+  setListingArray,
+  setPageNum,
+  setNumPerPage
 } from "../../actions/ListingDetail";
-// import {apartment1} from "../../resources/apartment1.jpg";
 import apartment1 from "../../resources/apartment1.jpg";
+import axios from "axios";
+import Paper from '@material-ui/core/Paper';
+import BathtubIcon from '@material-ui/icons/Bathtub';
+import HotelIcon from '@material-ui/icons/Hotel';
+import LocalLaundryServiceIcon from '@material-ui/icons/LocalLaundryService';
+import PetsIcon from '@material-ui/icons/Pets';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import Divider from '@material-ui/core/Divider';
+import Tooltip from '@material-ui/core/Tooltip';
+import { Typography } from "@material-ui/core";
+import Pagination from '@material-ui/lab/Pagination';
+
+import "../../styles/Listing.css";
 
 class Listing extends Component {
-  
+
   componentDidMount (){
-    this.props.setBedRoom(2);
-    this.props.setWashRoom(10);
+    // axios.get("http://localhost:3005/apartment1")
```

Comment:
Usually not good practice to commit commented-out code.

Reply:
we should, this is for post request which is for the rest API, as long as we don't have commented code in master during the end of an iteration, I think it should be fine.